### PR TITLE
Add support for ignored_tables and ignored_schema

### PR DESCRIPTION
**Pull request by abrarsheikh**
inspired from #173

### CODE REVIEW

File: pymysqlreplication/event.py [hunk 1]
```diff
@@ -7,11 +7,19 @@
...
+        event_size,
+        table_map,
+        ctl_connection,
+        only_tables=None,
+        ignored_tables=None,
+        only_schemas=None,
+        ignored_schemas=None,
+        freeze_schema=False,
+        fail_on_table_metadata_unavailable=False
+    ):
```
Comment (baloo) [inline pymysqlreplication/event.py]: I'm sorry to be picky but iI kind of dislike this style, could you please avoid making the diff too large and stick to the old style (which looks pep8 compliant besides the space around the operators)?  Same comment for `binlogstream.py` and `packet.py`.

**Comment by abrarsheikh** [comment]
@baloo thanks for the quick code review. Could you also bump up the package version :)

**Comment by baloo** [comment]
Cc @noplay

Sadly I don't have access to the pypi myself

**Comment by noplay** [comment]
I'm on it

**Pull request merged**